packages/edit-site/src/components/header/style.scss

```diff
@@ -7,14 +7,16 @@
 
 	.edit-site-header_start,
 	.edit-site-header_end {
-		flex: 1;
+		flex: 1 0 335px;
```

Comment:
what does this magic value do? it might be nice to add a comment about why it's needed :)

Reply:
Ah good point. It prevents the left and right toolbar from collapsing when shrinking the viewport (what Marko linked to earlier)

<img width="389" alt="Screenshot 2020-10-12 at 18 23 58" src="https://user-images.githubusercontent.com/1182160/95770402-a1ebcb80-0cb9-11eb-9302-81dbf1d668e1.png">

Reply:
I'll add an inline comment soon 👍